q2_metadata/normalization/_norm_errors.py

```diff
@@ -0,0 +1,204 @@
+# ----------------------------------------------------------------------------
```

Comment:
We should have a call about this module, looks like some of the classes here are repeating the work from _norm_check_rules.

Reply:
whenever you want

Reply:
we did have a fruitful call - thanks